### PR TITLE
refactor(context-menu): change to use script setup

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skipping review is not a problem (```style```, ```chore``` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
  - [ ] These changes require a usage change
- [x] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] Updated Storybook documents for component
- [ ] Wrote test codes for composable, utils and etc.
- [x] Tested with console(if usages are changed) 

### Description
SSIA

### CODE REVIEW

**Review by piggggggggy — APPROVED** [review]
LGTM :)

File: src/inputs/context-menu/PContextMenu.vue [hunk 1]
```diff
@@ -175,200 +172,170 @@ const getFilteredSelectedItems = (selected: MenuItem[], menu: MenuItem[]): MenuI
...
+    selected: () => [],
+    multiSelectable: false,
+    showRadioIcon: false,
+    strictSelectMode: false,
+    itemHeightFixed: false,
+    highlightTerm: '',
+    noSelectIndication: false,
+    showSelectHeader: false,
+    showClearSelection: false,
+    searchable: false,
```
Comment (yuda110) [inline src/inputs/context-menu/PContextMenu.vue]: question: Does `boolean` type also need a default value? Is it because of `undefined`?

Comment (WANZARGEN) [reply src/inputs/context-menu/PContextMenu.vue]: I removed it! thanx!

**Pull request merged**